src/main/flight/pid.c

```diff
@@ -842,6 +840,83 @@ void FAST_CODE applySmartFeedforward(int axis)
 }
 #endif // USE_SMART_FEEDFORWARD
 
+#if defined(USE_ABSOLUTE_CONTROL)
+static void applyAbsoluteControl(const int axis, const float gyroRate,
+    const float setpointLpf, const float setpointHpf,
+    float *currentPidSetpoint, float *itermErrorRate)
+{
+    if (acGain > 0) {
+        float acCorrection = 0;
+        float acErrorRate;
+
+        const float gmaxac = setpointLpf + 2 * setpointHpf;
+        const float gminac = setpointLpf - 2 * setpointHpf;
+        if (gyroRate >= gminac && gyroRate <= gmaxac) {
+            float acErrorRate1 = gmaxac - gyroRate;
+            float acErrorRate2 = gminac - gyroRate;
+            if (acErrorRate1 * axisError[axis] < 0) {
+                acErrorRate = acErrorRate1;
+            } else {
+                acErrorRate = acErrorRate2;
+            }
+            if (fabsf(acErrorRate * dT) > fabsf(axisError[axis]) ) {
+                acErrorRate = -axisError[axis] / dT;
+            }
+        } else {
+            acErrorRate = (gyroRate > gmaxac ? gmaxac : gminac ) - gyroRate;
+        }
+        acErrorRate = *itermErrorRate;
```

Comment:
All above computations are dead code as this line discards them

Reply:
Yes I know :D. Absolute control had even more dead code as it was being run even when disabled.

To me it seems a bit of a mess as it is now

Reply:
OK dead code is fixed now